test/JDBC/expected/BABEL-UNSUPPORTED.out

```diff
@@ -2950,3 +2950,32 @@ GO
 ~~ERROR (Message: 'EXTERNAL NAME' is not currently supported in Babelfish)~~
 
 
+-- create login from windows
+CREATE LOGIN [NT Service\MSSQLSERVER] FROM WINDOWS
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: 'NT Service' domain is not yet supported in babelfish)~~
+
+
+CREATE LOGIN [NT Service\MSSQLSERVER] FROM WINDOWS WITH DEFAULT_DATABASE=[test]
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: 'NT Service' domain is not yet supported in babelfish)~~
+
+
+CREATE LOGIN [NT Service\MSSQLSERVER] FROM WINDOWS WITH DEFAULT_DATABASE=[test], DEFAULT_LANGUAGE=[us_english]
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: 'NT Service' domain is not yet supported in babelfish)~~
+
+
+CREATE LOGIN [NT SerViCe\MSSQLSERVER] FROM WINDOWS WITH DEFAULT_DATABASE=[test], DEFAULT_LANGUAGE=[us_english]
```

Comment:
As discussed offline, let's remove `DEFAULT_LANGUAGE=[us_english]` since it's not supported yet. Please file a JIRA for the same.

Reply:
Removed DEFAULT_LANGUAGE from tests